api-tests/features/p1-no-photo-id.feature

```diff
@@ -298,3 +298,57 @@ Feature: P1 No Photo Id Journey
     Then I get a 'page-ipv-identity-postoffice-start' page response with context 'lastChoice'
     When I submit a 'end' event
     Then I get a 'pyi-escape' page response
+
+  Scenario: P1 thin file for low confidence users without photo ID
```

Comment:
I think this is already covered by `P1 DCMAW after KBV dropout Journey` in p1-journeys.feature ?

Reply:
The key difference here is that we are specifying activity score 0 whereas that test uses an activity score of 1. however they do give the same result